common/src/main/java/io/druid/concurrent/Execs.java

```diff
@@ -40,21 +40,41 @@ public static ExecutorService singleThreaded(String nameFormat)
     return Executors.newSingleThreadExecutor(makeThreadFactory(nameFormat));
   }
 
+  public static ExecutorService singleThreaded(String nameFormat, Integer priority)
+  {
+    return Executors.newSingleThreadExecutor(makeThreadFactory(nameFormat, priority));
+  }
+
   public static ExecutorService multiThreaded(int threads, String nameFormat)
   {
     return Executors.newFixedThreadPool(threads, makeThreadFactory(nameFormat));
```

Comment:
can we have those call the overloaded method instead of duplicating it, e.e `multiThreaded(int threads, String nameFormat, null)`